spec/formatting.md

```diff
@@ -85,10 +85,16 @@ At a minimum, it includes:
 
 Implementations MAY include additional fields in their _formatting context_.
 
-## Expression and Markup Resolution
+## Resolved Values
 
-_Expressions_ are used in _declarations_, _selectors_, and _patterns_.
-_Markup_ is only used in _patterns_.
+This specification allows for the same value to be used for:
```

Comment:
```suggestion
This specification allows for the same variable reference to be used in multiple parts of a _message_:
```

Reply:
Would that be clearer, though? Variable references are the mechanism that's used, by the key point here is that it's the same _value_ that's being used for multiple different purposes.

Reply:
I think these are saying kind of the same thing. The thing that's getting us into trouble is an (apparently?) slippery meaning of the word "value".

Reply:
Right, on first read it may seem like "value" means "resolved value". It's complicated by placeholders not being expressions. Otherwise you could say "allows for identical expressions to be used for:"

Reply:
This text is no longer present in the PR.